test-suite/Makefile

```diff
@@ -748,3 +749,23 @@ tools/%.log : tools/%/run.sh
 	    $(FAIL); \
 	  fi; \
 	) > "$@"
+
+# vos/
+
+vos: vos/run.log
+
+vos/run.log: $(wildcard vos/*.sh) $(wildcard vos/*.v)
```

Comment:
Shouldn't this be a misc/ test? It looks more special than it is when it lives in its own directory.

Reply:
(or maybe a coq-makefile/ test)